.github/workflows/tpv2.yml

```diff
@@ -16,53 +16,206 @@
+          --health-cmd="sh -c \"[[ \$(wget -qO- http://smtp/healthz) == true ]]\""
+          --
+          maildev/maildev:2.0.0-beta3
+          --smtp=25
+          --hide-extensions=STARTTLS
+          --web=80
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Cache Alpine Docker image
+        uses: actions/cache@v3
+        with:
+          path: ${{ github.workspace }}/docker-images
+          key: docker-images/alpine@${{ env.ALPINE_VERSION }}.tar.gz
+      - name: Import cached Alpine Docker image
+        run: .github/actions/save-alpine-tar/entrypoint.sh load ${{ env.ALPINE_VERSION }}
+      - name: Cache node modules
+        uses: actions/cache@v3
+        with:
+          path: ./experimental/traffic-portal/node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('./experimental/traffic-portal/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-modules-
+      - name: Initialize Traffic Ops Database
+        id: todb
+        uses: ./.github/actions/todb-init
+      - name: Initialize Traffic Vault Database
+        id: tvdb
+        uses: ./.github/actions/tvdb-init
+      - name: Check Go Version
+        run: echo "::set-output name=value::$(cat GO_VERSION)"
+        id: go-version
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ steps.go-version.outputs.value }}
+      - name: Build Traffic Ops
+        run: |
+          cd "${GITHUB_WORKSPACE}/traffic_ops/traffic_ops_golang"
+          go build .
+
+      # Setup
+      - name: Install dependencies
+        run: sudo apt-get update && sudo apt-get install google-chrome-stable postgresql-client gettext-base
+      - name: Create admin user
+        run: |
+          psql -c "INSERT INTO tm_user (username, role, tenant_id, local_passwd) VALUES ('admin', 1, 1, 'SCRYPT:16384:8:1:p0Bppp/6IBeYxSwdLuYddsdMLBU/BNSlLY6fSIF7H1XW4eTbNVeMPVm7TuTEG4FM8PbqLlVwi8sPy8ZJznAlaQ==:sRcHWGe43mm/uEmXTIw37GcLEQZTlWAdf4vJqK8f0MDh8P+8gXoNx+nxWyb3r/0Bh+yyg0g/dUvti/ePZJL+Jw==');"
+      - name: Create SSL Certificates and AES key
+        run: |
+          openssl rand 32 | base64 | tee "${GITHUB_WORKSPACE}/aes.key"
+          openssl req -new -x509 -nodes -newkey rsa:4096 -out traffic_ops/traffic_ops_golang/localhost.crt -keyout traffic_ops/traffic_ops_golang/localhost.key -subj "/CN=tptests"
+      - name: NPM install
+        run: |
+          cd experimental/traffic-portal
+          npm install
+      - name: Run everything and test
+        uses: ./.github/actions/tpv2-integration-tests
+      - name: Upload Report
+        uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: ${{ github.job }}
+          path: |
+            traffic_ops/traffic_ops_golang/out.log
+            experimental/traffic-portal/nightwatch/screens
+            experimental/traffic-portal/logs
+            experimental/traffic-portal/tests_output
```

Comment:
`nightwatch/junit` is not uploaded. I don't think it actually provides any additional info but it's being generated so we might as well include it in the artifacts.